docs/release_notes/v0.14.0.rst

```diff
@@ -0,0 +1,71 @@
+New in 0.14.0 (2021-10-04)
+--------------------------
+
+In this release, we have completely revamped the scvi-tools documentation website by creating a new set of user guides that provide: 
+
+a. The math behind each method (in a succinct, online methods-like way) 
+b. The relationship between the math and the functions associated with each model 
+c. The relationship between math variables and code variables 
+
+Our previous User Guide guide has been renamed to Tutorials and contains all of our existing tutorials (including tutorials for developers).
+
+Another noteworthy addition in this release is the implementation of the (amortized) Latent Dirichlet Allocation (aka LDA) model applied to single-cell gene expression data. We have also prepared a tutorial that demonstrates how to use this model, using a PBMC 10K dataset from 10x Genomics as an example application.
+
+Lastly, in this release we have made a change to reduce user and developer confusion by making the previously global ``setup_anndata`` method a static class-specific method instead. This provides more clarity on which parameters are applicable for this call, for each model class. Below is a before/after for the SOLO and TOTALVI model classes:
+
+.. figure:: figures/setup_anndata_before_after.svg
```

Comment:
for solo, most users will not call setup anndata because they use an alternative constructor that does it for them. I would pick another model. Also, the image is nice but you can just put code directly in this document

Reply:
Ah good call on Solo